Mathlib/Data/Sym/Sym2.lean

```diff
@@ -268,6 +268,16 @@ def _root_.Function.Embedding.sym2Map (f : α ↪ β) : Sym2 α ↪ Sym2 β wher
   toFun := map f
   inj' := map.injective f.injective
 
+lemma lift_comp_map {g : γ → α} (f : {f : α → α → β // ∀ a₁ a₂, f a₁ a₂ = f a₂ a₁}) :
```

Comment:
This lemma is backwards, since in general things should be named after their LHS. Can you flip the equality here and before?

Reply:
Done, and in `lift_apply_map`.